index.js

```diff
@@ -0,0 +1,159 @@
+let dataObject;//Declare object that contains a single key. Key's value is an array with words.
```

Comment:
Copy the value into an array rather than keeping the key value pair. It's not needed anymore. Also, rename to something more readable. Simply 'words'?